casanovo/casanovo.py

```
@@ -465,10 +465,15 @@ def setup_model(
     # Download model weights if these were not specified (except when
     # training).
     cache_dir = Path(appdirs.user_cache_dir("casanovo", False, opinion=False))
+
     if model is None:
         if not is_train:
             try:
-                model = _get_model_weights(cache_dir)
+                logger.warning(
```

Ruff (0.15.9)

[error] Too many arguments for `logging` format string

(PLE1205)